deny.toml

```diff
@@ -0,0 +1,11 @@
+[licenses]
```

Comment:
Is the intent here to impose Embark Studio’s deny configuration onto Prost?

Reply:
This isn't "our" deny configuration, this is just a way to ensure that the dependency this PR is removing doesn't return in the future.